modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/search/SearchService.kt

```diff
@@ -0,0 +1,14 @@
+package au.com.shiftyjelly.pocketcasts.servers.search
+
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface SearchService {
+    @GET("/autocomplete/search")
+    suspend fun autoCompleteSearch(
+        @Query("q") query: String,
+        @Query("termsLimit") terms: Int,
+        @Query("podcastsLimit") podcasts: Int,
```

Comment:
❓ Could we use here for parameter names also the full names? Like instead of `terms` `termsLimit`? `autoCompleteSearch(terms = 5, podcasts = 6)` doesn't read clearly.